main.nf

```diff
@@ -229,15 +264,20 @@ process rnaseqc {
     file '*.splice_events.{txt,pdf}' into results               // junction_annotation
     file '*.splice_junction.{txt,pdf}' into results             // junction_annotation
     file '*.junctionSaturation_plot.{txt,pdf}' into results     // junction_saturation
-    file '*.inner_distance.{txt,pdf}' into results              // inner_distance
+    file '*.inner_distance.*' into results              // inner_distance
```

Comment:
I think I got errors about missing files even though both were actually generated. This solved it. Add a comment to specify the files?